apps/emqx_auth_mnesia/src/emqx_acl_mnesia_cli.erl

```diff
@@ -156,12 +154,10 @@ cli(["add", "_all", Topic, Action, Access]) ->
     end;
 
 cli(["show", "clientid", Clientid]) ->
-    [emqx_ctl:print("Acl(clientid = ~p topic = ~p action = ~p access = ~p)~n",[NClientid, Topic, Action, Access])
-     || {{clientid, NClientid}, Topic, Action, Access, _} <- lookup_acl({clientid, iolist_to_binary(Clientid)}) ];
+    [print_acl(Acl) || Acl <- lookup_acl({clientid, iolist_to_binary(Clientid)})];
```

Comment:
use `lists:foreach(fun print_acl/1, ...)` here and below.
The list-comprehension expression produces a list dialyzer will complain.

Reply:
Using `lists: foreach (fun print_acl/1, ...)` causes the test case to fail, and the list-comprehension that there is no problem, so I chose the list-comprehension